src/crud.py

```diff
@@ -317,26 +317,30 @@ def check_calls_per_month(db, contract_id):
         db, contract_id)
     return max_calls >= len(nb_operations_already_made)
 
+# Function to limit the number of calls for new users
+# rather than for a specific sponsee.
+
```

Comment:
Please use docstrings to document functions instead, when it makes sense

Reply:
put it in the doc string at this [commit](https://github.com/marigold-dev/gas-station/pull/52/commits/d7063083fea97074120ca4309d857da92149491a)